multiqc/utils/search_patterns.yaml

```diff
@@ -535,6 +535,11 @@ ngsderive/encoding:
 ngsderive/junction_annotation:
   contents: "File	total_junctions	total_splice_events	known_junctions	partial_novel_junctions	complete_novel_junctions	known_spliced_reads	partial_novel_spliced_reads	complete_novel_spliced_reads"
   num_lines: 1
+nonpareil:
+  - fn: "*.json"
+    contents: "LRstar"
+    num_lines: 500
```

Comment:
We try to avoid making MultiQC loading long files into memory, is there any chance `num_lines` can be smaller? In the test example, `LRstar` is found on the line 25